src/Snicco/Component/better-wp-hooks/README.md

```diff
@@ -102,13 +102,13 @@ use Snicco\Component\BetterWPHooks\WPEventDispatcher;
 
 $dispatcher = WPEventDispatcher::fromDefaults();
 
-// Assumes MyListener has an __invoke method
+// Assumes OrderListener has an __invoke method
 $dispatcher->listen(OrderCreated::class, OrderListener::class);
 
-// string names work for events
+// String names work for events
 $dispatcher->listen('order_created', OrderListener::class);
 
-// Any public method works
+// Any public static method works
```

Comment:
🤔

Reply:
Public is correct. Objects will be created first from the container then the metbod is called.

Public static methods should work too but nobody should know that.😀

Reply:
I see, so classes are instantiated.